src/mlpack/core/data/impute_strategies/custom_strategy.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file custom_strategy.hpp
+ * @author Keon Kim
+ *
+ * Definition and Implementation of the empty CustomStrategy class.
+ */
+#ifndef MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_STRATEGY_HPP
+#define MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_STRATEGY_HPP
+
+#include <mlpack/core.hpp>
+
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+class CustomStrategy
+{
+  // empty class
+};
+
+} // namespace data
+} // namespace mlpack
+
```

Comment:
it is used for calling a overload of [Imputer::impute()](https://github.com/keonkim/mlpack/blob/5a517c25ef55de1f4814dc3605190d17f868ff82/src/mlpack/core/data/imputer.hpp#L86), which lets users to define a number that takes place of the missing value.

It basically does nothing but to fill the place of the template parameter.
It is used like [this](https://github.com/keonkim/mlpack/blob/5a517c25ef55de1f4814dc3605190d17f868ff82/src/mlpack/methods/preprocess/preprocess_imputer_main.cpp#L101). but I was not clear how to make this clean.